src/main/kotlin/no/nav/klage/oppgave/service/distribusjon/KlagebehandlingDistribusjonService.kt

```diff
@@ -35,19 +35,19 @@ class KlagebehandlingDistribusjonService(
                     logger.debug("Vedtak ${vedtak.id} i klagebehandling $klagebehandlingId er ikke distribuert")
 
                     klagebehandling = lagBrevmottakere(klagebehandling, vedtak)
-
-                    vedtak.brevmottakere
+                    val nyttVedtak = klagebehandling.getVedtak(vedtak.id)
+                    nyttVedtak.brevmottakere
```

Comment:
Synes dette var litt tricky kode. Ser problemet. Lurer på om du burde kalt `nyttVedtak` for `oppdatertVedtak` (eller `vedtakMedOppdaterteBrevmottakere`) i stedet? Kanskje man hadde skjønt litt mer av grunnen til å skrive koden slik da?

Reply:
Ja, enig i at navnet kan være mer forklarende, dette er litt debug-rester. Går for mest mulig eksplisitt, endrer til `vedtakMedOppdaterteBrevmottakere`.